src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.css

```
@@ -0,0 +1,24 @@
+.permissionChoice {
+  display: flex;
+  align-items: center;
+  padding: 10px 0;
+  border-bottom: 1px solid rgba(54, 54, 54, 0.15);
+  color: var(--temp-grey-blue-7);
+}
+
+.permissionChoiceDescription {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding-left: 15px;
+  font-size: var(--size-tiny);
+  line-height: 18px;
+
+  & span {
```

As I've noted in your other PR, while this is technically functional right now, it might not be in the future, so move this nested selector to it's own declration:
```suggestion
.permissionChoiceDescription span {
  ...
}
```